<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="task1">
     <attribute name="title">
      <string>Зазание 1</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>171</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>вычисление по формуле</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>211</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>задайте через пробел три числа</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ответ: </string>
      </property>
     </widget>
     <widget class="QPushButton" name="resultButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>174</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>calculate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="task2">
     <attribute name="title">
      <string>Задание 2</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="resultButton_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>174</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>calculate</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ответ: </string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>введите заначения</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>результат подстановки</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="task3">
     <attribute name="title">
      <string>Задание 3</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>значение i:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>значение x:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>161</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>состояние выполнения:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>вычислить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>190</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ответ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>190</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="task4a">
     <attribute name="title">
      <string>задание 4а</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>331</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>настройки для переменных</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>выбранная переменная</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>141</width>
         <height>25</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>x</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>y</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>z</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>80</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>применить</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>50</y>
         <width>71</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>значение</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>значения переменных</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>x:</string>
       </property>
      </widget>
      <widget class="QLabel" name="xvalue">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>y:</string>
       </property>
      </widget>
      <widget class="QLabel" name="yvalue">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>120</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>120</y>
         <width>16</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>z:</string>
       </property>
      </widget>
      <widget class="QLabel" name="zvalue">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>41</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>214</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>подсчитать</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>230</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ответ:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>230</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="task4b">
     <attribute name="title">
      <string>задание 4б</string>
     </attribute>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>16</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>n</string>
      </property>
     </widget>
     <widget class="QLabel" name="nValue">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>21</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>данные вычислений</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>311</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>вычислить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>результат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>230</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>значение x:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>398</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
